### DISCUSSION TITLE
Imported videos with no thumbnail and failing when playing

### DISCUSSION THREAD

**Discussion by antvio81** [photoprism/photoprism#1275] [Q&A] [answered]
Hi photoprism team,

First of all, I wanted to say well done on this product for photo management. I am new to it and only started using it recently with docker and found it really so user-friendly compared to others that are there out on the market. As being playing around only for few weeks, I have noticed a strange behaviour which I wanted to share with you and understand if this is normal or if there is something wrong with my configuration. So, here is my configuration:
- I have mounted a volume containing all the originals pictures. As such, the settings is with READ-ONLY Mode and also my mounted directory is with :ro. This is to prevent possible structure changes and make sure I keep the original one;
- I have imported the pictures and videos and run the index on the _originals_ folder; Everything seems fine.
- However, I noticed that in _Videos_  there are quite a few of them that don't have a thumbnail generated and not sure why this happened. The latter however, doesn't bother me. The problem I am facing is with playing the videos. When I try to play them some play immediately and fine and others, on the other hand, are just spinning and nothing happens. From the logs, I see the below message:
`level=info msg="converting XXX/XXX/XXXX/XXX.AVI to avc (libx264)"`

The above message is displayed with the majority of videos I am trying to play. 

I am not sure if this is the right place to post this but any help would be much appreciated. 

Thank you in advance and keep up the good work. 

Antonio

**Comment by rickysarraf** [comment, marked as answer]
I had the same problem. The thing is with newer H265 video format, which most browsers will not play. As I understand it, the future support for HEVC/H265 is bleak as it is encumbered with patents and royalties.

My easy approach was to re-encode them to H264, which is readily supported by the entire ecosystem. After that, PhotoPrism is happy too as it doesn't have to do any on-the-fly transcoding, which is very expensive.

**Reply by lastzero** [reply]
You may run "photoprism convert" in a terminal to pre transcode.
